cicd/scripts/runners/ValidateReleaseNotesRunner.ts

```diff
@@ -0,0 +1,492 @@
+	 * @returns The name of the label.
+	 */
+	private getPullRequestIncludeLabel(): string {
+		const prLabelRepoVarName = "PR_INCLUDE_NOTES_LABEL";
+
+		return this.vars.find((v) => v.name == prLabelRepoVarName)?.value ?? "";
+	}
+
+	/**
+	 * Gets all of the issues that are in a milestone with a title that matches the given {@link milestoneTitle},
+	 * excluding any issues that have a label that are in the ignore label list.
+	 * @param repoName The name of the repository.
+	 * @param milestoneTitle The title of the milestone.
+	 * @returns The issues in the milestone.
+	 */
+	private async getMilestoneIssues(repoName: string, milestoneTitle: string): Promise<[IIssueModel[], IIssueModel[]]> {
+		const ignoredIssues: IIssueModel[] = [];
+		const ignoreLabels: string[] = await this.getIgnoreLabels(repoName);
+
+		// Filter out any issues that have a label included in the ignore label list
+		const issues = (await this.milestoneClient.getIssues(repoName, milestoneTitle))
+			.filter((issue: IIssueModel) => {
+				if (ignoreLabels.length <= 0) {
+					return true;
+				}
+
+				const shouldNotIgnore = ignoreLabels.every((ignoreLabel) =>
+					issue.labels?.every((issueLabel) => issueLabel.name != ignoreLabel) ?? true
+				);
+				const shouldIgnore = !shouldNotIgnore;
+
+				if (shouldIgnore) {
+					ignoredIssues.push(issue);
+				}
+
+				return shouldNotIgnore;
+			});
+
+		return [issues, ignoredIssues];
+	}
+
+	/**
+	 * Gets all of the pull requests that are in a milestone with a title that matches the given {@link milestoneTitle},
+	 * and that have a label that matches the given {@link prIncludeLabel}.
+	 * @param repoName The name of the repository.
+	 * @param milestoneTitle The title of the milestone.
+	 * @param prIncludeLabel The label on a pull request to include the pull request in the release notes.
+	 * @returns The pull requests that are in the milestone.
+	 */
+	private async getMilestonePullRequests(
+		repoName: string,
+		milestoneTitle: string,
+		prIncludeLabel: string,
+	): Promise<[IPullRequestModel[], IPullRequestModel[]]> {
+		const ignoredPullRequests: IPullRequestModel[] = [];
+		const ignoreLabels: string[] = await this.getIgnoreLabels(repoName);
+
+		const pullRequests = (await this.milestoneClient.getPullRequests(repoName, milestoneTitle))
+			.filter((pr: IPullRequestModel) => {
+				const shouldNotIgnore = ignoreLabels.length <= 0 || ignoreLabels.every((ignoreLabel) => {
+					return pr.labels?.every((prLabel) => prLabel.name != ignoreLabel) ?? true;
+				});
+				const shouldIgnore = !shouldNotIgnore;
+
+				if (shouldIgnore) {
+					ignoredPullRequests.push(pr);
+				}
+
+				return (pr.pull_request?.merged_at != null && shouldNotIgnore) &&
+					pr.labels?.some((label) => label.name === prIncludeLabel);
+			});
+
+		return [pullRequests, ignoredPullRequests];
+	}
+
+	/**
+	 * Validates that the given list of {@link labels} exist in the repository.
```

Comment:
Refactor code docs to:

```suggestion
	 * Validates that the given list of {@link labels} exist in a repository with a name that matches the given {@link repoName}.
```